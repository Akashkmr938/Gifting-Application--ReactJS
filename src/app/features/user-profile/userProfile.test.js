import React from "react";
import renderer from "react-test-renderer";
import UserProfile from "./userProfile";

describe("The UserProfile component", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<UserProfile />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
