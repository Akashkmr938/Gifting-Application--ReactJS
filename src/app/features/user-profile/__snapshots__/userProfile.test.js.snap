// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The UserProfile component renders correctly 1`] = `
<div>
  <img
    alt="user profile"
    className="img"
    src=""
  />
  <h2>
    Name: 
    
  </h2>
  <h2>
    E-mail: 
    
  </h2>
  <h2>
    Wallet Balance: 
  </h2>
</div>
`;
